custom_components/luxtronik/const.py

```diff
@@ -275,21 +275,42 @@ class LuxParameter(StrEnum):
     P0002_DHW_TARGET_TEMPERATURE: Final = "parameters.ID_Einst_BWS_akt"
     P0003_MODE_HEATING: Final = "parameters.ID_Ba_Hz_akt"
     P0004_MODE_DHW: Final = "parameters.ID_Ba_Bw_akt"
-    P0011_HEATING_CIRCUIT_CURVE1_TEMPERATURE: Final = "parameters.ID_Einst_HzHwHKE_akt"
-    P0012_HEATING_CIRCUIT_CURVE2_TEMPERATURE: Final = "parameters.ID_Einst_HzHKRANH_akt"
-    P0013_HEATING_CIRCUIT_CURVE_NIGHT_TEMPERATURE: Final = (
+    # luxtronik*_heating_curve*
+    P0011_HEATING_CURVE_END_TEMPERATURE: Final = "parameters.ID_Einst_HzHwHKE_akt"
+    P0012_HEATING_CURVE_PARALLEL_SHIFT_TEMPERATURE: Final = "parameters.ID_Einst_HzHKRANH_akt"
+    P0013_HEATING_CURVE_NIGHT_TEMPERATURE: Final = (
         "parameters.ID_Einst_HzHKRABS_akt"
     )
-    # luxtronik*_heating_circuit2_curve*
-    P0014_HEATING_CIRCUIT2_CURVE1_TEMPERATURE: Final = (
+    # luxtronik*_heating_curve_circuit1*
+    P0014_HEATING_CURVE_CIRCUIT1_END_TEMPERATURE: Final = (
         "parameters.ID_Einst_HzMK1E_akt"  # 260
     )
-    P0015_HEATING_CIRCUIT2_CURVE2_TEMPERATURE: Final = (
+    P0015_HEATING_CURVE_CIRCUIT1_PARALLEL_SHIFT_TEMPERATURE: Final = (
         "parameters.ID_Einst_HzMK1ANH_akt"  # 290
     )
-    P0016_HEATING_CIRCUIT2_CURVE_NIGHT_TEMPERATURE: Final = (
+    P0016_HEATING_CURVE_CIRCUIT1_NIGHT_TEMPERATURE: Final = (
         "parameters.ID_Einst_HzMK1ABS_akt"  # 0
     )
+    # luxtronik*_heating_curve_circuit2*
+    P0017_HEATING_CURVE_CIRCUIT2_END_TEMPERATURE: Final = (
+        "parameters.ID_Einst_HzMK2E_akt"  # 260
+    )
+    P0018_HEATING_CURVE_CIRCUIT2_PARALLEL_SHIFT_TEMPERATURE: Final = (
+        "parameters.ID_Einst_HzMK2ANH_akt"  # 290
+    )
+    P0019_HEATING_CURVE_CIRCUIT2_NIGHT_TEMPERATURE: Final = (
+        "parameters.ID_Einst_HzMK2ABS_akt"  # 0
+    )
+    # luxtronik*_heating_curve_circuit3*
+    P0020_HEATING_CURVE_CIRCUIT3_END_TEMPERATURE: Final = (
```

Comment:
[P0020 is defined as "ID_Einst_BwTDI_akt_MO".](https://github.com/Bouni/python-luxtronik/blob/7a0eda74067caee4946fd624bc18a0eb32d43df8/luxtronik/parameters.py#L67)
Is P0020-P0022 really correct?
I think it should be P0774-P0776